shaded-deps/pom.xml

```diff
@@ -251,6 +253,10 @@
                             <pattern>org.jboss.netty</pattern>
                             <shadedPattern>org.apache.storm.shade.org.jboss.netty</shadedPattern>
                         </relocation>
+                        <relocation>
```

Comment:
Is the org.jboss.netty rule needed still?

Reply:
Zookeeper depends on Netty 3. I think we still need to relocate it, even though our code doesn't use it directly.

Reply:
That's right I remember now